unit-tests-jest/numbers.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers: number[]): number[] {
+  return numbers.filter((n) => n % 2 === 0);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ */
+export function toDollars(amount: number): string {
+  if (amount < 0) return `-$${(-amount).toFixed(2)}`;
+  return `$${amount.toFixed(2)}`;
+}
+
+/**
+ * Returns a new array of numbers where every entry has been divided by the
+ * given divisor. Does not modify the original array.
+ */
+export function divideBy(numbers: number[], divisor: number): number[] {
+  const dividedNumbers = numbers.map((num) => {
+    if (divisor === Infinity || divisor === 0) return Infinity;
```

Comment:
This test isn't strictly necessary since JavaScript will return Infinity for you.